cql/ScreeningSymptomaticLibrary.cql

```diff
@@ -119,20 +117,27 @@ define MostRecentNilmCytology:
   Dash.MostRecentNilmCytology
 
 define HasNilmCytologyWithinLast12Months:
-  Exists(Common.LookBack(Dash.MostRecentNilmCytology, 12 months))
+  Exists(Common.LookBack({Dash.MostRecentNilmCytology}, 12 months))
```

Comment:
Instead of casting this to a list to force it to work with `Exists`, could we do something like this to model it similarly to how we do these lookbacks for other singletons (for example in the `MostRecentCoincidentNilmCytologyAndNegativeHpvReport` expression below)?

```suggestion
  Dash.MostRecentNilmCytology C
  where Common.DiagnosticReportDate(C) within 12 months of Today()
  return
    C is not null
```

Reply:
We're just looking for a true/false here -- is the most recent Nilm cytology within the past 12 months? Would this be the same logic:
`Common.DiagnosticReportDate(Dash.MostRecentNilmCytology) >= Today() - 12 months`

Reply:
ok, we'll go with it